.ci/scripts/test_model.sh

```diff
@@ -150,10 +137,10 @@ test_model_with_xnnpack() {
   if [[ "${BUILD_TOOL}" == "buck2" ]]; then
     buck2 run //examples/xnnpack:xnn_executor_runner -- --model_path "${OUTPUT_MODEL_PATH}"
   elif [[ "${BUILD_TOOL}" == "cmake" ]]; then
-    if [[ ! -f ${CMAKE_OUTPUT_DIR}/backends/xnnpack/xnn_executor_runner ]]; then
-      build_cmake_xnn_executor_runner
+    if [[ ! -f ${CMAKE_OUTPUT_DIR}/executor_runner ]]; then
+      build_cmake_executor_runner
```

Comment:
You will have to make sure `EXECUTORCH_BUILD_XNNPACK` is ON, right? Maybe let `build_cmake_executor_runner` take a backend string and turn the option on accordingly? This can be extended to other backends.